controllers/userController.js

```diff
@@ -68,5 +68,19 @@ const deleteUserData = async (req, res) => {
     res.status(500).send({ message: err })
   }
 }
+const limitUsersData = async (req, res) => {
+  const page = req.params.page
+  const limitValue = Number(req.query.limit) || 10
+  try {
+    const userData = await userModel.findAndCountAll({ limit: limitValue, offset: (page - 1) * limitValue })
+    if (userData.rows.length != 0) {
+      res.status(200).send(userData)
+    } else {
+      res.status(500).send({ message: "oops data not found" })
```

Comment:
do not send 500 status without error, 500 is for there is some issue in code, that server can't handle.
So this response will also be 200 with count 0 and user data blank array @ashok-choudhary-0